bin/check-fail.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Checks tests introduced in pull requests fail on master.
+
+set -eo pipefail
+
+if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
+  echo "Skipping $0 for non-PR build"
+  exit 0
+fi
+
+if echo "${TRAVIS_PULL_REQUEST_BRANCH}" | grep -qi refactor; then
+  echo "The title of the PR indicates this is a refactoring; skipping this check"
```

Comment:
I know this is lifted wholesale from elsewhere, but you should probably fix the mismatch here – it's checking against the branch name, not the PR title.